### PR TITLE
[TASK] Split up TCEFORM

### PR DESCRIPTION
**Pull request by linawolf**
It has become quite long and hard to handle.

No text changes where made

releases: main, 12.4

### CODE REVIEW

File: Documentation/PageTsconfig/TceForm.rst [hunk 1]
```diff
@@ -17,53 +17,13 @@ disable options, blind options in selector boxes etc.
 See the core API document section :ref:`FormEngine <t3coreapi:FormEngine>` for more
 details on how records are rendered in the backend.
 
-..  _tceform-apply-properties:
+Read more on how to :ref:`Apply TCEFORM properties <tceform-apply-properties>`.
 
-Applying properties
-===================
+..  toctree::
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/TceForm.rst:22]: I suggest to remove the toctree:

tceform

The applying properties is doubled (sentence above) and the suggest looks displaced. It is linked at the end in the appropriate section. So no need for this toctree IMHO.

File: Documentation/PageTsconfig/TceForm/ApplyingProperties.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
+
+..  _tceform-apply-properties:
+
+===================
+Applying properties
+===================
+
+The properties listed below apply to various contexts which are explained per
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/TceForm/ApplyingProperties.rst:8]: "listed below" is not correct.

File: Documentation/PageTsconfig/TceForm/Suggest.rst [hunk 3]
```diff
@@ -0,0 +1,324 @@
+..  include:: /Includes.rst.txt
+..  index::
+    Records; Suggest wizard
+    Suggest wizard
+..  _pagetceformsuggest:
+
+suggest
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/TceForm/Suggest.rst:7]: The heading levels in this section are not correct.

**Comment by brotkrueml** [comment]
And actually, it looks half-baken to me. Maybe move all the properties to separate pages? Then you can also jump from the main menu to the appropriate property.

File: Documentation/PageTsconfig/TceForm.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/PageTsconfig/TceForm.rst:25]: I made a headline and also listed subtitles as the properties where moved to a subpage. This way still all properties can be seen on this page

File: Documentation/PageTsconfig/TceForm/ApplyingProperties.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/PageTsconfig/TceForm/ApplyingProperties.rst]: linked them instead

**Comment by linawolf** [comment]
I moved the properties to their own page

**Comment by linawolf** [comment]
@brotkrueml should we merge it like this or abandon the PR?

**Review by brotkrueml — APPROVED** [review]
I am not convinced, as "Suggest" holds also properties. But when you think, this is the way, go for it.

Additionally, I thought we would like to avoid classes (rubric h2)? I assume, this is rendered visually like a h2 but semantically it is not. So, not really a benefit to use rubric over a heading.

**Comment by linawolf** [comment]
I will abondon this. If you or someone else feels like it give it another, fresh try

**Pull request closed without merging**